charts/flame-hub/README.md

```
@@ -2,52 +2,74 @@
-  externalURL: "https://harbor.hub.local/" # don't forget https://
+```yaml
+auth:
+  existingSecret: "my-custom-secret"
+```
+The difference is that `existingSecret` is not populated with values from the chart or auto-generated. All credentials have to be provided beforehand.
+
+### Creating the secret
 
+The chart requires a secret with the following keys:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)